lib/saleor-app.ts

```diff
@@ -1,17 +1,12 @@
 import { SaleorApp } from "@saleor/app-sdk";
-import { FileAPL, VercelAPL } from "@saleor/app-sdk/APL";
+import { FileAPL, UpstashAPL, VercelAPL } from "@saleor/app-sdk/APL";
 
-const isVercel = process.env.VERCEL === "1";
-
-/**
- * For local development store auth data in the `.auth-data.json`.
- * For app deployment on Vercel with Saleor CLI, use vercelAPL.
- *
- * To read more about storing auth data, read the
- * [APL documentation](https://github.com/saleor/saleor-app-sdk/blob/main/docs/apl.md)
- */
-const apl = isVercel ? new VercelAPL() : new FileAPL();
+const mapping = {
+  FileAPL: new FileAPL(),
+  VercelAPL: new VercelAPL(),
+  UpstashAPL: new UpstashAPL(),
```

Comment:
cc @krzysztofwolski this is different than solution in app-template. Technically its ok, but maybe we can do something to unify, at least in apps Saleor owns

Reply:
Yes, I'll update it here and klavio after merging change in the template

Reply:
thanks for pointing this out - done 😄